Dalamud/Plugin/Internal/PluginManager.cs

```diff
@@ -535,6 +535,7 @@ async Task LoadPluginsAsync(string logPrefix, IEnumerable<PluginDef> pluginDefsL
     /// Reload all loaded plugins.
     /// </summary>
     /// <returns>A task.</returns>
+    [Obsolete("This method should no longer be used and will be removed in a future release.")]
```

Comment:
Maybe worth making this throw a compile-time error just to _really_ drive it home that people should not be touching this? Won't break existing plugins but it will stop new builds from working until people fix things.